java/dataflow-connector-examples/pom.xml

```diff
@@ -74,11 +74,14 @@ limitations under the License.
        <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
        <version>${beam.version}</version>
     </dependency>
+
+    <!-- [START bigtable_dataflow_connector_dep] -->
     <dependency>
         <groupId>com.google.cloud.bigtable</groupId>
         <artifactId>bigtable-hbase-beam</artifactId>
         <version>${bigtable.version}</version>
```

Comment:
Should a specific version be here?

Reply:
Hm.  For this case, I think that you're right that specific versions should be within the tag.

Reply:
Done